apps/shipments/models/permission_link.py

```diff
@@ -18,7 +18,6 @@
 from datetime import datetime, timezone
 
 from django.db import models
-
```

Comment:
Unnecessary change

Reply:
I think this was caused when I originally had the url_shortener on the model and optimized the imports.